backend/masterScript_createMapLayers.py

```diff
@@ -0,0 +1,78 @@
+# Import data sources information
```

Comment:
this file should be renamed in `create_map_layers.py`

Reply:
done